libtransmission/variant.cc

```diff
@@ -819,45 +820,36 @@ static void nodeDestruct(struct SaveNode* node)
  */
 void tr_variantWalk(tr_variant const* v_in, struct VariantWalkFuncs const* walkFuncs, void* user_data, bool sort_dicts)
 {
-    int stackSize = 0;
-    int stackAlloc = 64;
-    struct SaveNode* stack = tr_new(struct SaveNode, stackAlloc);
+    auto stack = std::vector<WalkNode>{};
```

Comment:
If it's a stack, why is it not a `std::stack<WalkNode>` (or `std::stack<WalkNode, std::vector<WalkNode>>` if you insist on using a vector as an impl)?

Reply:
stacked in 6f3547b